### DISCUSSION TITLE
Deactivate command set me back to regular zsh

### DISCUSSION THREAD

**Discussion by adrien3d** [ohmyzsh/ohmyzsh#12607] [Help] [answered]
By mistake, I have executed the command `deactivate` , and now, my terminal is the regular zsh.
How to set back by default omz?

**Comment by KianAnbarestani** [comment, marked as answer]
You can simply restart your shell so that your ~/.zshrc runs again and loads Oh My Zsh. For an immediate refresh, run:
`exec zsh`
If that doesn’t work, verify that your ~/.zshrc still sources Oh My Zsh (look for a line like `source $ZSH/oh-my-zsh.sh`).
